src/docs/references/page-schema/index.md

```diff
@@ -0,0 +1,1097 @@
+---
+title: Page schema
+slug: page-schema
+updated: 2023-02-15
+category: references
+ingress:
+  Reference documentation providing information on the page schema.
+published: true
+---
+
+> how the data looks like after people add fields and then remove them -
+> aka how the deleted data looks like in the fetched asset, as that’s
+> causing work for me. - And we should probably document those somewhere
+> for customizers.
+
+// TODO do we show the \_editor attributes as well?
+
+- The page schema is based on [JSON schema](https://json-schema.org/) –
+  [learn more](https://json-schema.org/understanding-json-schema/)
+- not to be confused with extended data search schema [TODO: LINK]
+- or web schema
+
+## Page Schema is divided into two parts
+
+- properties and defs
+- properties describe the structure of the page asset
+- defs contain subschemas, i.e. additional attributes that can be reused
+  in several properties
+  - read more about
+    [defs in JSON schema](https://json-schema.org/understanding-json-schema/structuring.html#defs)
+  - e.g. internal button link
+
+```json
+{
+  /** Section call to action */
```

Comment:
Hmm, would comments within JSON be misleading as they're not valid syntax?

Reply:
I was thinking about the same thing. For the draft, I included them to improve readability, since this version of the schema is human-readable only. If there's a better way to annotate the JSON within the article, I'd be happy to receive ideas!

Reply:
Removed comments from JSON, let's see if we get feedback on this being difficult to read or not 🤷‍♀️